Sources/Imperial/Services/Auth0/Auth0.swift

```diff
@@ -0,0 +1,26 @@
+import Vapor
+
+// https://auth0.com/docs/flows/guides/auth-code/add-login-auth-code
```

Comment:
If this is to be code docs can you `///` it? Might be worth expanding it as well to say why they'd want to reference that page